quickstart/java-redisson/redis-redisson-test/src/main/java/example/demo/App.java

```diff
@@ -0,0 +1,39 @@
+package example.demo;
+
+import org.redisson.Redisson;
+import org.redisson.api.RMap;
+import org.redisson.api.RedissonClient;
+import org.redisson.config.Config;
+
+import java.time.LocalDateTime;
+
+/**
+ * Redis test
+ *
+ */
+public class App {
+    public static void main(String[] args) {
+        // Connect to the Azure Cache for Redis over the TLS/SSL port using the key
+        Config config = new Config();
+        config.useSingleServer().setPassword(System.getenv("REDISCACHEKEY"))
+                .setAddress(String.format("rediss://%s:6380", System.getenv("REDISCACHEHOSTNAME")));
+        RedissonClient client = Redisson.create(config);
+
+        // Perform cache operations using the cache client
+        RMap<String, String> cache = client.getMap("namedCache");
+
+        // Simple get and put of string data into the cache
+        System.out.println("\nCache Command  : GET Message");
+        System.out.println("Cache Response : " + cache.get("Message"));
+
+        System.out.println("\nCache Command  : SET Message");
+        System.out.println("Cache Response : " + cache.put("Message",
+                String.format("Hello! The cache is working from Java! %s", LocalDateTime.now())));
+
+        // Demonstrate "SET Message" executed as expected
+        System.out.println("\nCache Command  : GET Message");
+        System.out.println("Cache Response : " + cache.get("Message"));
+
+        System.exit(0);
```

Comment:
In https://github.com/redisson/redisson-examples/blob/master/objects-examples/src/main/java/org/redisson/example/objects/SSLExamples.java, I noticed there's a call to `RedissonClient.shutdown()` instead of `System.exit(0)`. Is that appropriate here too?

```suggestion
        client.shutdown();
        System.exit(0);
```

From <https://github.com/redisson/redisson/wiki/16.-FAQ#q-when-do-i-need-to-shut-down-a-redisson-instance-at-the-end-of-each-request-or-the-end-of-the-life-of-a-thread>, it sounds like calling `shutdown()` isn't necessary when the app is exiting anyway, so probably this isn't necessary. Does that sound right?

Reply:
I tested the following cases:
* Just removing `System.exit(0);` and it turns out that the process won't exit;
* Call `client.shutdown();` at the end will successfully exit the process.

So I prefer to your suggested changes and committed.

Reply:
BTW, once `client.shutdown();` is called, the process will successfully exit, so `System.exit(0);` is no longer needed. Pushed commit 7f72e29 to get it removed.